firebase/functions/index.js

```diff
@@ -0,0 +1,31 @@
+const functions = require('firebase-functions');
+const nodemailer = require('nodemailer');
+const admin = require('firebase-admin');
+admin.initializeApp(functions.config().firebase);
+
+const gmailEmail = encodeURIComponent(functions.config().gmail.email);
+const gmailPassword = encodeURIComponent(functions.config().gmail.password);
+const mailTransport = nodemailer.createTransport(
+    `smtps://${gmailEmail}:${gmailPassword}@smtp.gmail.com`);
+
+const APP_NAME = 'AD.MYSPORTS';
+
+exports.sendWelcomeEmail = functions.auth.user().onCreate(event => {
+  const user = event.data; 
+  const email = user.email; 
+  const displayName = user.displayName; 
+  return sendWelcomeEmail(email, displayName,user);
+});
+
+function sendWelcomeEmail(email, displayName,user)   {
+  const mailOptions = {
+    from: `${APP_NAME} <noreply@firebase.com>`,
```

Comment:
for me the config.gmail.email appears on from field, so I imagine this line is useless